CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+For general contributing guidelines, see the [Contributing Guide on DoneJS.com][].
+
+## Developing Locally
+
+### Setup
+
+Fork and clone this repository. See [Fork A Repo][] and [Cloning a repository][] for help.
+
+Install the packages defined in [`package.json`][]:
+
+```
+npm install
+```
+
+### Build
+
+There is no build step for this repo.
+
+### Test
+
+Tests are located in `test.js`, at the root of this repo.
+
+To run the tests, execute the `test` script defined in [`package.json`][]:
+
+```
```

Comment:
Here’s the master list: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

I thought `shell` was the best option, but if you find a better one, we can start using that.